reconcile/ldap_users_api/integration.py

```
@@ -0,0 +1,362 @@
+
+logger = logging.getLogger(__name__)
+
+QONTRACT_INTEGRATION = "ldap-users-api"
+
+
+class LdapUsersApiIntegrationParams(PydanticRunParams):
+    """Parameters for LDAP users API integration."""
+
+    app_interface_repo_url: str
+    infra_repo_url: str
+    infra_paths: list[str] = []
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)